### ISSUE TITLE
meow-next does not move cursor down, but creates a untitled buffer and puts me there

### ISSUE THREAD

**Issue by greghab** [meow-edit/meow#465]
`meow-next`, or `n` in my keybindings case, is supposed to move the cursor down.
However, when I call that / hit the keybind, it creates a blank buffer named `untitled` (`untitled`, then `untitled<2>`, etc), and puts me there.
I'm just using the stock config for `dvp`, and I don't know why that is occuring. Any Ideas?
- (Moving the cursor left, right, up, and things like delete `d`, work fine. It's just this moving down, that it does this behavior).
``` elisp
(straight-use-package 'meow)
(defun meow-setup ()
  (setq meow-cheatsheet-layout meow-cheatsheet-layout-dvp)
  (meow-leader-define-key
   '("?" . meow-cheatsheet))
  (meow-motion-overwrite-define-key
   ;; custom keybinding for motion state
   '("<escape>" . ignore))
  (meow-normal-define-key
   '("?" . meow-cheatsheet)
   '("*" . meow-expand-0)
   '("=" . meow-expand-9)
   '("!" . meow-expand-8)
   '("[" . meow-expand-7)
   '("]" . meow-expand-6)
   '("{" . meow-expand-5)
   '("+" . meow-expand-4)
   '("}" . meow-expand-3)
   '(")" . meow-expand-2)
   '("(" . meow-expand-1)
   '("1" . digit-argument)
   '("2" . digit-argument)
   '("3" . digit-argument)
   '("4" . digit-argument)
   '("5" . digit-argument)
   '("6" . digit-argument)
   '("7" . digit-argument)
   '("8" . digit-argument)
   '("9" . digit-argument)
   '("0" . digit-argument)
   '("-" . negative-argument)
   '(";" . meow-reverse)
   '("," . meow-inner-of-thing)
   '("." . meow-bounds-of-thing)
   '("<" . meow-beginning-of-thing)
   '(">" . meow-end-of-thing)
   '("a" . meow-append)
   '("A" . meow-open-below)
   '("b" . meow-back-word)
   '("B" . meow-back-symbol)
   '("c" . meow-change)
   '("d" . meow-delete)
   '("D" . meow-backward-delete)
   '("e" . meow-line)
   '("E" . meow-goto-line)
   '("f" . meow-find)
   '("g" . meow-cancel-selection)
   '("G" . meow-grab)
   '("h" . meow-left)
   '("H" . meow-left-expand)
   '("i" . meow-insert)
   '("I" . meow-open-above)
   '("j" . meow-join)
   '("k" . meow-kill)
   '("l" . meow-till)
   '("m" . meow-mark-word)
   '("M" . meow-mark-symbol)
   '("n" . meow-next)
   '("N" . meow-next-expand)
   '("o" . meow-block)
   '("O" . meow-to-block)
   '("p" . meow-prev)
   '("P" . meow-prev-expand)
   '("q" . meow-quit)
   '("r" . meow-replace)
   '("R" . meow-swap-grab)
   '("s" . meow-search)
   '("t" . meow-right)
   '("T" . meow-right-expand)
   '("u" . meow-undo)
   '("U" . meow-undo-in-selection)
   '("v" . meow-visit)
   '("w" . meow-next-word)
   '("W" . meow-next-symbol)
   '("x" . meow-save)
   '("X" . meow-sync-grab)
   '("y" . meow-yank)
   '("z" . meow-pop-selection)
   '("'" . repeat)
   '("<escape>" . ignore)))
(require 'meow) ;; https://github.com/meow-edit/meow/issues/162
(meow-setup)
(meow-global-mode 1)
```

**Comment by DogLooksGood** [comment]
Hey, please check this https://github.com/meow-edit/meow/blob/master/GET_STARTED.org#must-read-before-you-start.

In this case, you should tweak variable `meow--kbd-forward-line`.

Meow assumes you would keep the default key bindings. Otherwise you have to tweak these variables.

**Comment by greghab** [comment]
Appreciate it. Resolved the issue.

----

I read in that section you linked to, that `C-w`, maps to `meow-kill`
Then I looked at my keyboard mappings of `   '("n" . meow-next)`, and I assumed it may be doing `C-n` or something (But I could have looked up that cursor down is defaulted to `C-n`, without needing to guess).
Anyway, without `meow` enabled, I did `C-n`, and saw that it did created that `untitled` buffer again, and sent me there.

Did a `M-x describe-key C-n`, and saw:
```
C-n runs the command xah-new-empty-buffer (found in global-map), which
is an interactive native-compiled Lisp function in ‘xah-fly-keys.el’.

It is bound to C-n, <f7> c n.
```
I didn't have `xah-fly-keys` enabled nor was I using it, but had a keymap set (`(xah-fly-keys-set-layout 'programer-dvorak)`). After removing that, it  got rid of that additional mapping to `C-n` (which was interfering with meow), and this behavior resolved itself.
